alf/environments/suite_dmlab_test.py

```diff
@@ -26,8 +25,6 @@ def setUp(self):
         super().setUp()
         if not suite_dmlab.is_available():
             self.skipTest('suite_dmlab is not available.')
-        else:
```

Comment:
Right now this test is skipped because dmlab is not installed. I suspect that removing this will make this test fail. I think a better way is removing gin.bind_parameter('torch.optim.Adam.params', [{'params': []}]) in alf/utils/external_configurable.py. We can add a class alf.optimizers.Adam derived from torch.optim.Adam with default value for `params`

Reply:
Add them back after Haichao's PR [508](https://github.com/HorizonRobotics/alf/pull/508)